doc/releases/changelog-0.26.0.md

```diff
@@ -4,27 +4,84 @@
 
 <h3>New features since last release</h3>
 
-<h4>QNSPSA optimizer 😤</h4>
+<h4>Classical shadows 👤</h4>
 
-* A new optimizer called `qml.QNSPSAOptimizer` is available that implements the quantum natural simultaneous
-  perturbation stochastic approximation (QNSPSA) method based on 
-  [Simultaneous Perturbation Stochastic Approximation of the Quantum Fisher Information](https://quantum-journal.org/papers/q-2021-10-20-567/). 
-  [(#2818)](https://github.com/PennyLaneAI/pennylane/pull/2818) 
+* All-new features for implementing the classical-shadows protocol are now available. 
+  [(#2820)](https://github.com/PennyLaneAI/pennylane/pull/2820)
+  [(#2821)](https://github.com/PennyLaneAI/pennylane/pull/2821)
+  [(#2871)](https://github.com/PennyLaneAI/pennylane/pull/2871)
+  [(#2968)](https://github.com/PennyLaneAI/pennylane/pull/2968)
+  [(#2959)](https://github.com/PennyLaneAI/pennylane/pull/2959)
+  [(#2968)](https://github.com/PennyLaneAI/pennylane/pull/2968)
 
-  `qml.QNSPSAOptimizer` can be viewed as a second-order SPSA algorithm. It requires 10 circuit 
-  executions per optimization step compared to 3 for `qml.SPSAOptimizer`.
-  The additional circuit executions are used to provide a stochastic estimation of a second-order
-  metric tensor, which often helps the optimizer to achieve faster convergence. 
+  The classical-shadow measurement protocol is described in detail in the
+  [classical shadows paper](https://arxiv.org/abs/2002.08953).
```

Comment:
```suggestion
  The classical-shadow measurement protocol is described in detail in the paper
  [Predicting Many Properties of a Quantum System from Very Few Measurements](https://arxiv.org/abs/2002.08953).
```

Optional, feel free to accept/edit/decline. It felt a bit informal referring to the paper as the classical shadows paper!